app/lib/actions/remote_execution/run_hosts_job.rb

```diff
@@ -27,7 +27,6 @@ def run(event = nil)
 
       def output
         result = super
-        result[:pending_count] = (result[:total_count] || 0) - (result[:failed_count] || 0) - (result[:success_count] || 0)
         result
```

Comment:
oops :-) removed the whole method, it's no longer necessary (after this was merged https://github.com/Dynflow/dynflow/commit/4c918383f1234c22decdc3e75d4e0b75af0c73c1)